workspaces/homepage/plugins/dynamic-home-page/src/components/EntitySection/EntitySection.tsx

```diff
@@ -0,0 +1,269 @@
+              <Grid
+                item
+                xs={12}
+                md={isRemoveFirstCard ? 3 : 3.5}
+                key={item.metadata.name}
+              >
+                <EntityCard
+                  link={`/catalog/default/${item.kind.toLocaleLowerCase('en-US')}/${item.metadata.name}`}
+                  title={item.spec?.profile?.displayName ?? item.metadata.name}
+                  version="latest"
+                  description={item.metadata.description ?? ''}
+                  tags={item.metadata?.tags ?? []}
+                  kind={item.kind}
+                />
+              </Grid>
+            ))}
+            {entities?.length === 0 && (
+              <Grid item md={isRemoveFirstCard ? 12 : 7}>
+                <Box
+                  sx={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    justifyContent: 'center',
+                    minHeight: 300,
+                    border: muiTheme =>
+                      `1px solid ${muiTheme.palette.grey[400]}`,
+                    borderRadius: 3,
+                    overflow: 'hidden',
+                  }}
+                >
+                  <CardContent>
+                    <div className={classes.title}>
+                      No software catalog added yet
+                    </div>
+                    <div className={classes.description}>
+                      Once software catalogs are added, this space will showcase
+                      relevant content tailored to your experience.
+                    </div>
+                    <Box>
+                      <Link
+                        to="/catalog-import"
+                        underline="none"
+                        className={classes.link}
+                      >
+                        <div
+                          style={{
+                            display: 'flex',
+                            justifyContent: 'center',
+                          }}
+                        >
+                          Register a component
+                        </div>
+                      </Link>
+                    </Box>
+                  </CardContent>
+                </Box>
+              </Grid>
+            )}
+          </Grid>
+          <Box sx={{ pt: 2 }}>
+            {entities?.length > 0 && (
+              <ViewMoreLink to={catalogEntityLink}>
+                View all {data?.totalItems ? data?.totalItems : ''} catalog
+                entities
+              </ViewMoreLink>
+            )}
+          </Box>
+        </React.Fragment>
+      </Box>
+    );
+  }
+
+  return (
+    <Box
+      component={Paper}
+      sx={{
+        padding: '24px 24px 0 24px',
+        border: muitheme => `1px solid ${muitheme.palette.grey[300]}`,
+        borderRadius: 3,
+      }}
+    >
+      <div
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+          fontSize: '1.5rem',
```

Comment:
Can we use a h2 or a Typography tag here?

This is visually a headline, then we should also use a markup that makes it more a11y.